elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -5,175 +5,167 @@
 import base64
 import os
 import json
+import math
 
 PL_ANSWER_CORRECT_DEFAULT = True
 PL_ANSWER_INDENT_DEFAULT = -1
 INDENTION_DEFAULT = False
+MAX_INDENTION_DEFAULT = 4
 SOURCE_BLOCKS_ORDER_DEFAULT = 'random'
 GRADING_METHOD_DEFAULT = 'ordered'
-MIN_INCORRECT_DEFAULT = None
-MAX_INCORRECT_DEFAULT = None
 SOURCE_HEADER_DEFAULT = 'Drag from here:'
-SOLUTION_HEADER_DEFAULT = 'Construct your solution here:'
+SOLUTION_HEADER_DEFAULT = 'Drop blocks here:'
```

Comment:
Why this change in wording?

Reply:
I thought "construct your solution" was a little bit vague and unclear, whereas if we had the two headers say "Drag blocks from here" and "Drop blocks here", it will be more clear to the students. What do you think?

Reply:
I like how the old version has the word 'solution' in it because it makes it clear that portion is what they are being graded on. But I'm not sure if it actually matters.

Reply:
Yeah, especially since we have implemented a help button... I guess we shouldn't fix what isn't broken? I haven't heard any student feedback since the `pl-order-blocks` element went live tho.